templates/home.html

```diff
@@ -0,0 +1,54 @@
+{% load staticfiles %}
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+        <title>Wikimedia - Pages Created and Edited Tool</title>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+  		<script src="https://code.jquery.com/jquery-1.12.4.js"></script>
```

Comment:
We try to avoid external assets. I suppose with Google Charts that's pointless since it will use the API anyway, but with the rest it should not be a problem.

Reply:
Done.